### DISCUSSION TITLE
concat dataset mmdet 3.0

### DISCUSSION THREAD

**Discussion by Chop1** [open-mmlab/mmdetection#9948] [Q&A]
Hello,

I'm confused with the concatDataset in 3.0

In particular with the val evaluator

```python
val_evaluator = dict(
    type='CocoMetric',
    ann_file=['A.json', 'B.json'],
    metric=['bbox', 'segm'],
    format_only=False)
```

Apparently, val evaluator does not support a list for ann_file. How to make it work with concatDataset then?

**Comment by stweigand97** [comment]
I am wondering the same exact thing, have you figured it out?

I've openend an issue #10822 😊
